workers/unity/Packages/com.improbable.gdk.core/Worker/DiffCreatorFromOpList.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using System.Collections.Generic;
+using Improbable.Worker.CInterop;
+
+namespace Improbable.Gdk.Core
+{
+    internal class DiffCreatorFromOpList
```

Comment:
If we really want it to be static it can be. That Id can be given to it as an argument. Originally this class did a lot more, and it could also be used to support things like hot reloading. But it can easily be made static.